smart-contracts/test/Lock/createLockWithInfiniteKeys.js

```diff
@@ -16,14 +17,16 @@ contract('Lock / createLockWithInfiniteKeys', () => {
   describe('Create a Lock with infinite keys', () => {
     let transaction
     before(async () => {
-      transaction = await unlock.createLock(
+      const args = [
         60 * 60 * 24 * 30, // expirationDuration: 30 days
-        web3.utils.padLeft(0, 40),
+        web3.utils.padLeft(0, 40), // token address
         web3.utils.toWei('1', 'ether'), // keyPrice: in wei
         constants.MAX_UINT, // maxNumberOfKeys
-        'Infinite Keys Lock',
-        '0x000000000000000000000000'
-      )
+        'Infinite Keys Lock', // name
+        // '0x000000000000000000000000',
```

Comment:
Do you know what this was?

Reply:
Oh that was the salt, correct? 
Do we want to keep it? I think the goal was to "pre-compute" the address of the deployed lock, but I don't know if we can do it with this new approach...

Reply:
Ok after offline convo I think we can safely get rid of the hashes :)
We will have to see how the lock proxies are deployed to see if we can compute their address before deployment but that's a front-end issue that we should solve there.